src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory SESSION_FACTORY = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        return SESSION_FACTORY;
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
```

Comment:
Ensure that the `hibernate.cfg.xml` file is present in your project and correctly configured, as it is essential for the `SessionFactory` initialization.